backend/btrixcloud/colls.py

```diff
@@ -797,13 +797,13 @@ async def list_urls_in_collection(
         crawl_ids = await self.get_collection_crawl_ids(coll_id)
 
         match_query: dict[str, object] = {"oid": oid, "crawl_id": {"$in": crawl_ids}}
-        sort_query: dict[str, int] = {"count": -1, "_id": 1}
```

Comment:
This sorting was a fix by Sua to make the thumbnail/home page selection sort order reasonable, I don't think we want to change it here as it'll negatively impact the frontend.